apps/synapse-portal-framework/src/components/DetailsPage/DetailsPage.tsx

```diff
@@ -117,6 +119,7 @@ export default function DetailsPage(props: DetailsPageProps) {
         rowData: row,
       }}
     >
+      <DetailsPageDocumentMetadata resourcePrimaryKey={resourcePrimaryKey} />
```

Comment:
For all DetailsPage instances, add document metadata (internally, nothing will happen if `resourcePrimaryKey` is undefined)